src/views/router_views/index/home.njk

```diff
@@ -2,10 +2,24 @@
 {% from "govuk/components/button/macro.njk" import govukButton %}
 
 {% block main_content %}
-      <h1 class="govuk-heading-xl">{{ "Update a registered company email address - NOT FINAL" }}</h1>
+      <h1 class="govuk-heading-xl">{{ "Update a registered email address" }}</h1>
       <p class="govuk-body-l">Use this service to update a registered company email address.</p>
-      <p class="govuk-body">You cannot use this service to register a company email address.</p>
-      <p class="govuk-body">Instructions about doing that via a <a href="https://www.gov.uk/government/publications/etc/etc/etc" target="_blank" class="govuk-link">confirmation statement</a> here.</p>
+      <p class="govuk-body">You cannot use this service if you are providing a registered email address for the first time for a company. 
```

Comment:
Looking at the prototype, line `6` should read:

`Use this service to update a company's registered email address.`

Reply:
<img width="562" alt="image" src="https://github.com/companieshouse/registered-email-address-web/assets/123395274/72abc233-1e4f-4a8b-8dff-dd03b9292a96">